### ISSUE TITLE
Add a deployment tutorial

### ISSUE THREAD

**Issue by cretz** [rust-qt/ritual#52]
When doing `cargo run --release` on https://github.com/cretz/rust-qthello (with qmake and msvc 64 bit vars on path to properly build deps), I get the following on execution:

![image](https://user-images.githubusercontent.com/206396/27096394-84a64138-5036-11e7-8276-0cc8e74c4a83.png)

It should be noted that it all builds properly. Am I missing some DLLs I need to place in the output folder?

**Comment by cretz** [comment]
I have solved this, see the README at https://github.com/cretz/rust-qthello. I put the Windows subsystem on main which is important in the latest release for removing the console win in the background. Then I took the two dlls built by your libs and coupled them w/ 3 from the Qt bin dir adjacent to the exe for it to work.

My issue, based on my use of the dependency walker, was that I was using old Qt libs that were on my PATH from elsewhere. Here's what that simple prog looks like:

![image](https://user-images.githubusercontent.com/206396/27099062-4c30475a-503f-11e7-8cf1-224e63ca802d.png)

Note, you cannot use [windeployqt](http://doc.qt.io/qt-5/windows-deployment.html) because the exe is not recognized as a Qt app. Shall I leave this open pending some documentation on Windows deployment, or close as not a bug?
